server/web/public.js

```diff
@@ -15,6 +15,23 @@ exports.register = function(server, options, next) {
     }
   });
 
+  server.route({
+    method: "GET",
+    path: "/robots.txt",
+    handler: {
+      file: "public/robots.txt"
+    }
+  });
+
+  // TODO: should this be /public/ like non-precomposed?
```

Comment:
should this be `/public/apple-touch-icon-precomposed.png` like the non-precomposed icon? if you tell mobile safari where the one is, does it use that to try to find the other?